src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,82 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  setOption: (val: any) => void
+  option: string
+  query: string
+  setQuery: (val: any) => void
```

Comment:
We try not to use type "any" as we loose all the benefits of strict typing in typescript)
```suggestion
  setQuery: (val: string) => void
```